modules/update-mirror-repository-oc-mirror.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="update-mirror-repository-oc-mirror_{context}"]
+= Mirroring resources using the oc-mirror plug-in
+
+Using the oc-mirror OpenShift CLI (`oc`) plug-in is the recommended method for mirroring images onto a mirror registry. This can be useful in several situations, including fully or partially disconnected environments.
+
+.Procedure
+
+. Navigate to _Mirroring images for a disconnected installation using the oc-mirror plug-in_.
+
+. Follow the instructions on that page to mirror resources onto a mirror registry.
+
+** If you are using oc-mirror for the first time, follow the instructions on that page up until but not including the section titled *Keeping your mirror registry content updated*.
+
+** If you have already used oc-mirror to mirror resources onto a registry, follow the instructions in the section titled *Keeping your mirror registry content updated*.
```

Comment:
We don't normally use bold to refer to section titles.

I typically use italics, but the jury is out on what we should use and there is a mix of italics and double quotes. But you should use one of those instead of bold.